### PR TITLE
Add Unit support to Config

### PR DESCRIPTION
**Pull request by Hydrox6**
Adds an annotation (like `@Range`) to specify the units to be displayed in the box. This should lessen confusion on certain config items that have their units hidden by length or by being in the tooltip.
image
image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/config/Units.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Used with ConfigItem, defines what units are shown to the side of the box.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+@Documented
+public @interface Units
+{
+	String name() default "";
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/config/Units.java]: 1) why is there a default
2) if you make this `String value()` then you can just use `@Unit("ms")`

Comment (Hydrox6) [reply runelite-client/src/main/java/net/runelite/client/config/Units.java]: Ah, didn't know it worked like that.

**Comment by Hydrox6** [comment]
Updated to use a `UnitType` enum

**Comment by abextm** [comment]
I don't like the enum — It prevents external plugins from using this. It would probably be better to have some static final strings on the interface if we want to centralize parts of this.

**Comment by Hydrox6** [comment]
Now updated to use static strings (apparently the `public static final` was unneeded, according to IDEA)

**Pull request merged**